canonical-web-and-design/ubuntu.com
# Add /engage index

## Done

- created a very simple /enage index page for SEO

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/engage
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the page has a very simple list of all engage pages
